CONTRIBUTING.md

```diff
@@ -86,6 +86,64 @@ Once you downloaded the images you do the following for each screenshot:
 - If the change in the screenshot is **desired**, add the screenshot to the
   repo, commit it and push it to your PR's branch.
 
+## Pull Request Merging Policy
+Pull Requests (PRs) are reviewed by the
+[backend](https://github.com/orgs/cBioPortal/teams/backend),
+[frontend](https://github.com/orgs/cBioPortal/teams/frontend) and
+[devops](https://github.com/orgs/cBioPortal/teams/devops) teams of cBioPortal.
+For each PR the submitter should propose what team(s) is/are appropriate to
+review it. This is the current merging policy:
+
+- A documentation change needs one **+1**
+- A simple bugfix to rc or hotfix requires one **+1**
+- A new feature requires two **+1**
+
+If these requirements are met, any person with merge rights can merge to rc or
+hotfix.
+
```

Comment:
I think that for new features it is good to include a reviewer from another institute